src/TingBundle/TingBundle.php

```diff
@@ -28,5 +28,5 @@
 
 class TingBundle extends Bundle
 {
-    public const VERSION = '3.5';
+    public const VERSION = '3.8';
```

Comment:
This may be 3.10 ?

Reply:
This is actually the bundle version, so 3.8 is ok